cluster-setup/base/performance/operator_operatorgroup.yaml

```diff
@@ -3,6 +3,3 @@ kind: OperatorGroup
 metadata:
   name: performance-addon-operator
   namespace: openshift-performance-addon
-spec:
```

Comment:
are we sure this is needed? operatorgroup is about the OLM installation flow, not about the operator runtime behaviour.

Reply:
Will  check and get back, thanks!

Reply:
tested and it would not work:
Warning  UnsupportedOperatorGroup  36s   operator-lifecycle-manager  OwnNamespace InstallModeType not supported, cannot configure to watch own namespace

oc get csv -A
NAMESPACE                              NAME                                DISPLAY                      VERSION   REPLACES   PHASE
openshift-operator-lifecycle-manager   packageserver                       Package Server               0.16.1               Succeeded
openshift-performance-addon            performance-addon-operator.v4.6.0   Performance Addon Operator   4.6.0                Failed

Reply:
I see. This deserves a bit more followup then, because we need to see if OperatorGroup is still needed at all.

Reply:
Yep, not needed => removed. Thanks

Reply:
Actually -> deployment failed:
failed: error transitioning ClusterServiceVersion: csv in namespace with no operatorgroups

Reply:
So needed

Reply:
Yes, but we need to dig a bit deeper to review if empty spec is what we need. I guess I'll need to catch up with v1 changes.

Reply:
Yes, I double checked -> empty space => all namespaces

Reply:
https://docs.openshift.com/container-platform/4.5/operators/understanding/olm/olm-understanding-operatorgroups.html

"Alternatively, you can omit both spec.selector and spec.targetNamespaces to specify a global OperatorGroup, which selects all namespaces:"

Reply:
ok, will do a deeper review later. What you found makes sense. Still, feels like a bit awkward limitation/constraint of OLM.